### PR TITLE
Contributing Gerrit Plugin

### PR DESCRIPTION
**Pull request by andym0457**
#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [x] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Comment by benjdlambert** [comment]
Hey :wave:

I think that it probably makes more sense to create a `gerritAuthApiRef` like `githubAuthApiRef` to do the communication with the gerrit endpoints rather than having to setup a proxy for this?

We're looking at moving away from the proxy to favour direct communication or if it's more efficient, to create a corresponding backend plugin to do use the already built in `integrations` config that exists today: https://backstage.io/docs/integrations/gerrit/locations/

Basically want to copy some of how the `PullRequestCard` component get it's data from Github: https://github.com/backstage/backstage/blob/0870577bd4ac6922ff999a0207c2aed71512d499/plugins/github-pull-requests-board/src/api/useOctokitGraphQl.ts#L26

Does this make sense?

**Comment by andym0457** [comment]
Thanks @benjdlambert for the prompt feedback, I'll look at changing the implementation to work similar to the githubAuthApiRef

**Comment by benjdlambert** [comment]
Hey @andym0457, this looks like it got closed for staleness with no reply - just let me know when you're ready for it to be reopened! :tada:
